code/modules/religion/sparring/sparring_datum.dm

```diff
@@ -0,0 +1,293 @@
+/datum/sparring_match
+	///the chaplain. it isn't actually a chaplain all the time, but in the cases where the chaplain is needed this will always be them.
+	var/mob/living/carbon/human/chaplain
+	///the other fighter
+	var/mob/living/carbon/human/opponent
+	///what weapons will be allowed during the sparring match
+	var/weapons_condition
+	///area instance the participants must stay in
+	var/area/arena_condition
+	///what stakes the fight will have
+	var/stakes_condition
+	///cheats from the chaplain
+	var/chaplain_violations_allowed = 2
+	///cheats from the non-chaplain
+	var/opponent_violations_allowed = 2
+	///outside interventions that ruin the match
+	var/flubs = 2
+
+/datum/sparring_match/New(weapons_condition, arena_condition, stakes_condition, mob/living/carbon/human/chaplain, mob/living/carbon/human/opponent)
+	. = ..()
+	src.weapons_condition = weapons_condition
+	src.arena_condition = arena_condition
+	src.stakes_condition = stakes_condition
+	src.chaplain = chaplain
+	src.opponent = opponent
+	ADD_TRAIT(chaplain, TRAIT_SPARRING, TRAIT_GENERIC)
+	ADD_TRAIT(opponent, TRAIT_SPARRING, TRAIT_GENERIC)
```

Comment:
darn, I can't stack up 200 duels to try to one-shot the colossus anymore, and I also can't earn all 7 of the points I need for upgrades from one fight by just starting 7 duels against one guy and then finishing them all at once.

sad!